src/components/legend.ts

```diff
@@ -29,22 +29,22 @@ export module Components {
     /**
      * Creates a Legend.
      *
-     * The legend consists of a series of legend entries, each with a color and label taken from the `colorScale`.
-     * The entries will be displayed in the order of the `colorScale` domain.
+     * The legend consists of a series of legend entries, each with a color and label taken from the `scale`.
```

Comment:
Mise capitalize `Legend` while you're at it, and remove the word "legend" from in front of "entries"